optimum/habana/diffusers/pipelines/ddpm/pipeline_ddpm.py

```diff
@@ -29,6 +29,8 @@
 from optimum.habana.transformers.gaudi_configuration import GaudiConfig
 from optimum.utils import logging
 
+from ....utils import speed_metrics
```

Comment:
@deepak-gowda-narayana For better readability and maintainability, could you please update this to use absolute paths instead?

Reply:
it seems though all stable_difussion pipeline code is using like this.

Reply:
Yes, that is correct and we should update those in future

Reply:
Done

Reply:
I would rather keep relative imports in source code for a few reasons:
- While absolute imports are clearer, they are also more ambiguous regarding what is actually imported. I've seen a few broken environments where absolute imports find a wrong version of the package that is installed somewhere and that leads to errors that are quite complicated to debug.
- Consistency with other HF libraries, as this is the way we do it in all our libraries.
- Relative imports are slightly faster, even though that's not critical at all for us.

I think both approaches have their strengths and weaknesses but I would like to keep relative imports because of the first two points I listed above.

Reply:
@deepak-gowda-narayana apologize for the confusion. Based on regiss explanation, would you please revert to the original relative import as you did. Thanks